at-quannguyen/W4/main.rb

```diff
@@ -0,0 +1,59 @@
+require './bbcnews.rb'
+require './cnnnews.rb'
+require './user.rb'
+
+bbc_news_1 = BbcNews.new(1, 'BBC first news', 'Hello world from BBC', 'Say hello')
+bbc_news_2 = BbcNews.new(2, 'BBC second news', 'Hello Ruby from BBC', 'Say hello')
+bbc_news_3 = BbcNews.new(3, 'BBC third news', 'Hello AsianTech from BBC', 'Say hello')
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/a7fd7c22501ceb429d470c5982e714f3d16f651e/at-quannguyen/W4/main.rb#L7"></span>Naming/VariableNumber: Use normalcase for variable numbers.</td>
    </tr>
  </tbody>
</table>